### PR TITLE
Added Github section in Devops resources

### LINKED ISSUE #425: 💡 [FEATURE] -  Add GitHub Repository Section in DevOps section
### Idea Contribution

- [X] I have read all the feature request issues.
- [X] I'm interested in working on this issue
- [X] I'm part of GSSOC organization

### Explain feature request

In DevOps Resources , there is no GitHub repository link , where someone will explore how it works , I want to add this feature.

### Explain your solution

I want to add a section called GitHub repository where i will share the link of many GitHub repo where someone will find many projects of DevOps , for beginners to advance level.

### Any alternative approaches/features

_No response_

### Additional Context
I want to contribute in this issue , will you assign this to me @jfmartinz .

### PR DESCRIPTION
**Pull request by Pujan-sarkar**
## - `DevOps- ` - GitHub Repository

Fixes #425 

### Checklist: 

- [X] I have carefully reviewed and adhered to the [contributing guidelines](https://github.com/jfmartinz/ResourceHub/blob/main/CONTRIBUTING.md) before creating this pull request.
- [X] I followed the prescribed PR title template. _(Check this if you're adding a resource)_

### CODE REVIEW

**Comment by Pujan-sarkar** [comment]
@jfmartinz  i have created new pr , check this time

File: DevOps/readme.md [hunk 1]
```diff
@@ -316,3 +317,48 @@ DevOps is a combination of software development (Dev) and operations (Ops). It i
       </tr>
       
 </table>
+
+### GitHub Repositories
+
+<table width="100%">
+    <tr>
+        <th>Resource</th>
```
Comment (jfmartinz) [inline DevOps/readme.md:325]:
```suggestion
        <th>Resource Name</th>

```

**Review by jfmartinz — CHANGES_REQUESTED** [review]

**Comment by Pujan-sarkar** [comment]
@jfmartinz  I CANT UNDERSTAND THE CHNAGES YOU WANT AND I HAVE ALREADY DONE IT , SO CAN YOU CHECK IT ONCE

**Comment by Pujan-sarkar** [comment]
@jfmartinz  please can you reply..

**Review by jfmartinz — APPROVED** [review]
Thanks @Pujan-sarkar.

**Comment by jfmartinz** [comment]
I hope you understand that I can quicly reply to the messages, I am the only one maining the project and also have school. 

> @jfmartinz please can you reply..

**Comment by jfmartinz** [comment]
Also the PR is less than a day

**Comment by Pujan-sarkar** [comment]
@jfmartinz sorry sir... For the disturbance

**Comment by jfmartinz** [comment]
No worries 👍

**Pull request merged**